src/modules/account/api/update-user.api.ts

```diff
@@ -0,0 +1,10 @@
+import { useMutation } from '@tanstack/react-query';
+import { type PersonalInfoData } from '../schemas/personal-info.schema';
+import { apiClient } from '#/shared/libs/api-client.lib';
+
+export function useUserUpdateMutation() {
+  const mutation = useMutation<void, Error, PersonalInfoData>({
+    mutationFn: (data: PersonalInfoData) => apiClient.put('/users/profile', data),
+  });
+  return { mutation };
```

Comment:
Please don't nest ```mutation```. You can return ```mutation``` itself (instead of ```{ mutation }```) and access ```mutation``` in the component as well

Reply:
Fixed